src/constants.js

```diff
@@ -0,0 +1,3 @@
+export const COLLECTIONS = {
+  USERS: "Users",
```

Comment:
For future reference the USER key word here, some people start going back to camel case. Don't worry about it for now.